packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,368 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from 'react-portal';
+import PopupContent from './_PopupContent';
+import PopupArrow from './_PopupArrow';
+import PopupOverlay from './_PopupOverlay';
+import TetherComponent from './_TetherComponent';
+import './Popup.scss';
+
+const MIRROR_LR = {
+  center: 'center',
+  left: 'right',
+  right: 'left',
+};
+
+const MIRROR_TB = {
+  middle: 'middle',
+  top: 'bottom',
+  bottom: 'top',
+};
+
+const BASE_WIDTH = 16;
+const BASE_HEIGHT = 9;
+
+const DIMENSIONS = [
+  '10x',
+  '25x',
+  '50x',
+  '75x',
+  '100x',
```

Comment:
Still waiting on final values from ux, but I coded it that way because they are coefficient multipliers of the base 16/9 object. So they refer to 25x16/10x9, etc. I don't really know what these will end up being. But I know a grid style sizes are in the works. I'm all ears for alternatives.